src/Components/Tasks/Tasks.tsx

```diff
@@ -54,79 +85,336 @@ const Tasks: React.FC  = () => {
-                  <td>User</td>
-                  <td>{task?.description}</td>  
-                  <td>
-                    <i
-                      // onClick={() => showViewModal(project?.id)}
-                      className="fa fa-eye  text-info px-2"
-                    ></i>
-                    <i
-                      // onClick={() => showUpdateModal(project)}
-                      className="fa fa-pen  text-warning px-2"
-                    ></i>
-                    <i
-                      // onClick={() => showDeleteModal(project.id)}
-                      className="fa fa-trash  text-danger"
-                    ></i>
-                  </td>
-                </tr>
-              ))
-            // ) : (
-            //   <NoData/>
-            // )
-            }
-          </tbody>
-        </table>
+      <div className="header d-flex justify-content-between p-3">
+        <h3>Tasks</h3>
+        <Link
+          to='/dashboard/tasks/add-task'
+          className="btn btn-warning rounded-5 px-4"
+        >
+          <i className="fa fa-plus" aria-hidden="true"></i> &nbsp;Add New
+          Task
+        </Link>
 
       </div>
+
+      {/* table */}
+      <>
+        <div className="table-container1 vh-100">
+          <table className="table">
+            <thead className="table-head table-bg ">
+              <tr>
+                <th scope="col">Title</th>
+                <th scope="col">Status</th>
+                <th scope="col">Description</th>
+                <th scope="col">User</th>
+                <th scope="col">Project</th>
+                <th scope="col">Date Created</th>
+                <th scope="col">Actions</th>
+              </tr>
+            </thead>
+            <tbody>
+              {
+                tasks?.length > 0 ? (
+                  tasks.map((task: any) => (
+                    <tr key={task?.id}>
+                      <th scope="row">{task?.title}</th>
+                      <td>{task?.status}</td>
+                      <td>{task?.description}</td>
+                      <td>{task.employee.userName}</td>
+                      <td>{task.project.title}</td>
+                      <td>{new Date(task.creationDate).toLocaleDateString()}</td>
+                      <td>
+                        <i
+                          onClick={() => showViewModal(task?.id)}
+                          className="fa fa-eye  text-info px-2"
+                        ></i>
+                        <i
+                          onClick={() => showUpdateModal(task)}
+                          className="fa fa-pen  text-warning px-2"
+                        ></i>
+                        <i
+                          onClick={() => showDeleteModal(task.id)}
+                          className="fa fa-trash  text-danger"
+                        ></i>
+                      </td>
+                    </tr>
+                  ))
+                ) : (
+                  <NoData />
```

Comment:
Consider moving the `<NoData/>` component outside of the table as it shouldn't be a child of a tbody `element`

![image](https://github.com/Rawan-Mohamed/project-management-system-1/assets/64327685/3f8a5eb0-7bf0-4968-b688-9269edee5113)